src/main/java/com/espro/flink/consul/checkpoint/ConsulCheckpointRecoveryFactory.java

```diff
@@ -30,7 +34,6 @@ public ConsulCheckpointRecoveryFactory(Supplier<ConsulClient> client, Configurat
 		this.configuration = Preconditions.checkNotNull(configuration, "configuration");
 	}
 
-	@Override
 	public CompletedCheckpointStore createCheckpointStore(JobID jobId, int maxNumberOfCheckpointsToRetain, ClassLoader userClassLoader) throws Exception {
```

Comment:
make this method `private` and remove unused parameter `userClassLoader`

Reply:
done